docs/reference/modules/cluster/remote-clusters-security.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[remote-clusters-security]]
+=== Configure remote clusters with security
+To use {ccr} or {ccs} with secured clusters, all clusters must have the {es} 
+security features enabled.
```

Comment:
I made some bigger changes about explaining security impacts in the main remote clusters page, but this language needs updating as well. I think that we want to inform users that security is really important with CCR/CCS, though it isn't actually _required_ (to my knowledge). That said, I think we can change this to read:
>To use {ccr} or {ccs} safely with remote clusters, enable security on all connected clusters and configure Transport Layer Security (TLS) on every node.

Reply:
Yes security is not required for remote clusters since CCS is a core feature. But we should strong recommend it. And also be explicit about "secured" means both configuring TLS (at least for transport) and RBAC (both requires `xpack.security.enabled: true`)